src/Input/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -5,17 +5,21 @@ import { SecondaryColor } from '../../Utils/Colors';
 import { Toggle, ToggleBall } from './Styles';
 
 export interface ToggleSwitchProps {
+  /** This will determine whether or not the component renders with an active or inactive state at the start. */
   defaultActive?: boolean;
+  /** The icon components that will be used in both active and inactive states. Both icons have to be defined when opting to use this. */
   iconOptions?: {
     active: React.ReactType;
     inactive: React.ReactType;
   };
+  /** Called when the state of the toggle has been changed. Returns true or false depending on whether the toggle is active or inactive. */
```

Comment:
```suggestion
  /** Called when the state of the toggle has been changed. Called with true or false depending on whether the toggle became active or inactive. */
```
Actually it's called with the boolean, it doesn't return it.